### DISCUSSION TITLE
Check out the latest data and stats on drought across the U.S.

### DISCUSSION THREAD

**Discussion by megan8617** [earthlab-education/Earth-Analytics-2023-01-Intro#172] [Resources]
Drought is pretty frequently on my mind, and I came across this excellent resource in a lecture last summer (before we got this past winter's rains here in California):

https://www.drought.gov/data-maps-tools/us-drought-monitor, 

which ultimately links to
https://droughtmonitor.unl.edu/

The "Data" tab allows you to select an area, region, or state, and filter for time periods and drought severity. The site also provides a nice metadata addition.

**Comment by eculler** [comment]
Out west we like to keep track of snow drought too, since some 80% of our water comes from snowpack.  My lab group in grad school had a tradition of checking out something like [these data (from SNOTEL and Snow Course)](https://www.nrcs.usda.gov/wps/portal/wcc/home/quicklinks/imap#version=167&elements=R&networks=!&states=!&counties=!&hucs=&minElevation=&maxElevation=&elementSelectType=all&activeOnly=true&activeForecastPointsOnly=false&hucLabels=false&hucIdLabels=false&hucParameterLabels=true&stationLabels=&overlays=&hucOverlays=state&basinOpacity=100&basinNoDataOpacity=100&basemapOpacity=30&maskOpacity=0&mode=data&openSections=dataElement,parameter,date,basin,options,elements,location,networks,baseMaps,overlays,labels&controlsOpen=true&popup=&popupMulti=&popupBasin=&base=esriWgc&displayType=station&basinType=6&dataElement=WTEQP&depth=-8&parameter=PCTMED&frequency=DAILY&duration=I&customDuration=1&dayPart=E&monthPart=E&forecastPubDay=1&forecastExceedance=50&useMixedPast=true&seqColor=1&divColor=7&scaleType=D&scaleMin=&scaleMax=&referencePeriodType=POR&referenceBegin=1981&referenceEnd=2010&minimumYears=20&hucAssociations=true&relativeDate=-1&lat=41.755&lon=-108.896&zoom=5.0) at the beginning of meetings.
